### PR TITLE
Path variable tests

### PR DESCRIPTION
**Pull request by matsl**
## What

Path variable tests.

### CODE REVIEW

File: test/hbut-tests.el [hunk 1]
```diff
@@ -194,5 +194,33 @@ the button text"
...
+
+(ert-deftest hbut-pathname-path-variable-with-two-colons-is-one-file-test ()
+  "Path variable with two colons is not seen as a path variable."
+  (with-temp-buffer
+    (insert "\"/var/lib:/bar:/tmp\"")
+    (goto-char 16)
+    (hy-test-helpers:action-key-should-call-hpath:find "/var/lib:/bar:/tmp")))
+
+(ert-deftest hbut-pathname-path-variable-with-three-colons-is-a-path-test ()
+  "Path variable with two colons is not seen as a path variable."
```
Comment (rswgnu) [inline test/hbut-tests.el:212]: Fix doc string here to be three colons.

File: test/hbut-tests.el [hunk 2]
```diff
@@ -194,5 +194,33 @@ the button text"
            (should (equal (car err) 'error))
            (should (string-search "ibutton at point already has a label" (cadr err)))))))))
 
+(ert-deftest hbut-pathname-path-variable-test ()
+  "Find file in path variable."
```
Comment (rswgnu) [inline test/hbut-tests.el:198]: Given that there are different Hyperbole actions when working with a 'path variable name' and a 'path variable value', I find it helpful to use those two terms everywhere so you know what type you are working with.

**Review by rswgnu — CHANGES_REQUESTED** [review]
Generally, looks good.  Will approve after small doc updates.  Need to test selecting paths with completions when selecting a path variable name at some point.  Thanks.

**Review by rswgnu — APPROVED** [review]

File: test/hbut-tests.el (hunk 2 above)
Comment (matsl) [reply test/hbut-tests.el:198]: Not sure about what is the best naming. I have always referred to PATH and its cousins as search paths or path lists. So for functions that actually take the `PATH` variable there `path variable` sounds good. Are you suggesting that for functions that takes the value that could come from a path variable we should call it `path variable value`? And not `path list` or `search path`? 

I'm happy to use whatever you suggest that is consistent with what we have in the code I just have not thought about it enough.

So in this case I should use the doc string `Find file in path variable value`?

**Pull request merged**